2019-Spring-WashU.md

```diff
@@ -387,10 +387,6 @@ Most class periods will be split between lecture/discussion about Open Source So
 ### Wed Apr 24
 * Conclusion of Final examination
 * Sprint 6 retro
-* Check-in:
```

Comment:
I'd prefer to have this check-in remain. By this point, the client should've seen the project a number of times to provide guidance. Doco as well as the final presentation should be nearly complete going into reading week as well.